### ISSUE TITLE
Custom operations missing in completion list

### ISSUE THREAD

**Issue by ronaldschlenker** [dotnet/fsharp#11578]
Given is a builder that defined a "body" custom operation. I expect that this "body" custom operation is shown in the completion list when being inside of a computation.

In the completion list shown below, the custom operation is not shown:

![image](https://user-images.githubusercontent.com/472222/118629479-e7332700-b7cd-11eb-83b8-9cd26c7b0474.png)

This is also *not* working:

![image](https://user-images.githubusercontent.com/472222/118630263-aab3fb00-b7ce-11eb-894c-1e721f6f6d0b.png)

In fact, the operation is present and recognized when it is used:

![image](https://user-images.githubusercontent.com/472222/118630652-04b4c080-b7cf-11eb-9aa2-42f1bd411c67.png)

Using a pre-defined builder instance, the custom operation "body" is shown in the completion list (as expected):

![image](https://user-images.githubusercontent.com/472222/118630045-73454e80-b7ce-11eb-8527-178d88b27f55.png)

---------------------------

Might be related to:

#4832

---------------------------

Microsoft Visual Studio Professional 2019
Version 16.9.5

**Comment by cartermp** [comment]
Yes, this is likely related to https://github.com/dotnet/fsharp/issues/4832 but I don't think it's the same issue. I imagine we're just not pulling in the names during name resolution when we should, at least not in all cases. Clearly it does work when consuming a builder you don't own in source.
